css-view-transitions-2/Overview.bs

```diff
@@ -205,7 +205,22 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		```
 	</div>
 
+# Pseudo-classes # {#pseudo-classes}
 
+## Active View Transition Pseudo-class '':active-view-transition()''' ## {#the-active-view-transition-pseudo}
+
+The <dfn id='active-view-transition-pseudo'>:active-view-transition(* | <<transition-types>>)</dfn> pseudo-class represents
+the root element of the document, if it has a matching [=active view transition=].
```

Comment:
Can you use semantic line breaks : https://rhodesmill.org/brandon/2012/one-sentence-per-line/. We went with that for the L1 spec.

Reply:
Done